### PR TITLE
[Docs](bangc-ops): Fixed the 0.5 doc _02_22

### PR DESCRIPTION
**Pull request by WangYuan-cn**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

Please briefly describe what modification is made in this pull request, and indicate where to make the modification.

## 3. Test Report

If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).

#### 3.1.1 Accuracy Acceptance Standard

For static threshold standard details, see: [MLU-OPS Accuracy Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Accuracy-Acceptance-Standard.md).

- [ ] diff1: diff1 <= 3e-3
- [ ] diff2: diff2 <= 3e-3

#### 3.1.2 Operator Scheme checklist

|     No.        |                 Details              |            Check Results             |
|----------------|--------------------------------------|--------------------------------------|
|        1       |Supported hardware                    |             MLU370<br>MLU590         |
|        2       |Job types                             |          block <br> U1 <br> U4       |
|        3       |Layouts                               |          NHWC 、NCHW、ARRAY etc      |
|        4       |Whether multi-dimensions are supported|                                      |
|        5       |Whether element zero is supported     |                                      |
|        6       |Data type(half/float)                 |           half / float etc           |
|        7       |Whether there is size limit           |                                      |

#### 3.1.3 New Feature Test

If you have checked the following items, please tick the relevant box.

- [ ] Data type test
- [ ] Multidimensional tensor test
- [ ] Layout test
- [ ] Different size/integer remainder end segment/alignment misalignment test
- [ ] Zero dimensional tensor test/zero element test
- [ ] stability test
- [ ] Multiple platform test
- [ ] Gen_case module test
- [ ] Nan/INF tests 
- [ ] Bug fix tests
- [ ] For memory leak check details, see[GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).
- [ ] For code coverage check details, see: [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).
- [ ] For I/O calculation efficiency check details, see: [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md).

#### 3.1.4 Parameter Check

When a new operator is submitted, the test points are given and the test results are stated.

|                   Test Point                    | Acceptance Standard | Test Result (Error Message) |
| ----------------------------------------------- | --------------------| --------------------------- |
| Whether it conforms to the operator restriction |     Normal error    |                             |
| Whether illegal parameters are passed           |     Normal error    |                             |

### 3.2 Accuracy Test

For the cases used in the New Feature Test section, the features and the number of cases are recorded here. When multiple operations are tested, multiple tables are needed to include details of these operations.

Operation:

|Test Point           | Description                      | Quantity |  Comment |
|----------           |----------------------------------|----------|  --------|
|Data type test       |half/float/int8                   |          |          |
|Mult-tensor test     |Supports 1-8 dims                 |          |          |
|Layout test          |Supports NCHW/NHWC                |          |          |
|Zero element test    |Whether to support this test      |          |          |
|Stability test       |--gtest_repeat=NUM<br>--thread=NUM|          |          |
|Mult-platform test   |MLU370/MLU590                     |          |          |
|Nan/INF test         |Whether to support this test      |          |          |

### 3.3 Performance Test

See [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md) for details.

Platform：MLU370

|Operation|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-------|----|----|-----|----|----|----|-----|
|op_name|    |    |     |    |    |    |     |
|op_name|    |    |     |    |    |    |     |

Platform：MLU590

|Operation|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-------|----|----|----|----|----|----|-----|
|op_name|    |    |    |    |    |    |     |
|op_name|    |    |    |    |    |    |     |

### 3.4 Summary Analysis

Please give a brief overview here, if you want to note and summarize the content.

### CODE REVIEW

File: bangc-ops/mlu_op.h [hunk 1]
```diff
@@ -4952,12 +5081,11 @@ mluOpExpand(mluOpHandle_t handle,
...
  * @par Example
  * - The example of the fill operation is as follows:
-     @verbatim
-      param:value: 5
-
-      output array by 2 * 3 * 2 --> output: [[[5,5],[5,5],[5,5]],
-                                             [[5,5],[5,5],[5,5]]]
-     @endverbatim
+ *   @verbatim
+ *    param:value: 5 *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:5085]: 1

File: bangc-ops/mlu_op.h [hunk 2]
```diff
@@ -4834,13 +4964,12 @@ mluOpBallQuery(mluOpHandle_t handle,
...
-     @verbatim
-      input array by 2 * 2
-      --> then: [[1, 8], [6, 4]]
-
-      output array by 2 * 2
-      --> output: [[1, 8], [6, 4]]
-     @endverbatim
+ *   @verbatim
+ *    input array by 2 * 2
+ *    --> then: [[1, 8], [6, 4]] *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:4969]: 2

File: bangc-ops/mlu_op.h [hunk 3]
```diff
@@ -2139,26 +2253,24 @@ mluOpAddN_v2(mluOpHandle_t handle,
...
-
-       Output tensor   :   [[3,  6,  9],
-                            [12, 15, 18],
-                            [21, 24, 27]]
-     @endverbatim
- *
+ *   @verbatim
+ *     Input tensor  1 :   [[1, 2, 3],
+ *                          [4, 5, 6],
+ *                          [7, 8, 9]] *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:2259]: 3

File: docs/bangc-docs/release_notes/bangc_ops.rst [hunk 4]
```diff
@@ -62,6 +62,8 @@ v0.5.0
 
    * ``active_rotated_filter_forward``
 
+   * ``add_n``
+
```
Comment (LittleReal) [inline docs/bangc-docs/release_notes/bangc_ops.rst:66]: 这里是否要统一为算子接口名

File: bangc-ops/mlu_op.h [hunk 5]
```diff
@@ -2061,25 +2178,22 @@ mluOpGetAddNWorkspaceSize(mluOpHandle_t handle,
...
-
-       Input num       :   3
-
-       Output tensor   :   [[3,  5,  7],
-                            [9, 11, 13],
-                            [15,17, 19]]
-     @endverbatim
- *
+ *   @verbatim
+ *     Input tensor  1 :   [[1, 2, 3]] *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:2182]: 4

File: bangc-ops/mlu_op.h [hunk 6]
```diff
@@ -5006,12 +5134,11 @@ mluOpFill(mluOpHandle_t handle, float value, const mluOpTensorDescriptor_t outpu
...
  * @par Example
  * - The example of the fill operation is as follows:
-     @verbatim
-      input array by 1 --> value: [1]
-
-      output array by 2 * 3 * 2 --> output: [[[5,5],[5,5],[5,5]],
-                                             [[5,5],[5,5],[5,5]]]
-     @endverbatim
+ *   @verbatim
+ *    input array by 1 --> value: [1] *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:5138]: 5

File: bangc-ops/mlu_op.h [hunk 7]
```diff
@@ -5068,12 +5195,11 @@ mluOpFill_v2(mluOpHandle_t handle,
...
  * @par Example
  * - The example of the fill operation is as follows:
-     @verbatim
-      param:value: 5
-
-      output array by 2 * 3 * 2 --> output: [[[5,5],[5,5],[5,5]],
-                                             [[5,5],[5,5],[5,5]]]
-     @endverbatim
+ *   @verbatim
+ *    param:value: 5 *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:5199]: 6

File: bangc-ops/mlu_op.h [hunk 8]
```diff
@@ -6370,36 +6525,34 @@ mluOpUniqueGetOutLen(mluOpHandle_t handle,
...
-    @endverbatim
+ *   @verbatim
+ *     Example 1:
+ *     input array:
+ *       input: [1, 1, 2, 4, 4, 9, 7, 8, 8]
+ *     param:
+ *       mode: MLUOP_UNSORT_FORWARD
+ *     output array:
+ *       output_data: [1, 2, 4, 9, 7, 8]
+ *       output_index: [0, 0, 1, 2, 2, 3, 4, 5, 5] *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:6536]: 7

File: bangc-ops/mlu_op.h [hunk 9]
```diff
@@ -6370,36 +6525,34 @@ mluOpUniqueGetOutLen(mluOpHandle_t handle,
...
+ *     Example 2:
+ *     input array:
+ *       input: [1, 1, 2, 4, 4, 9, 7, 8, 8]
+ *     param:
+ *       mode: MLUOP_SORT_ASCEND, return_inverse: true, return_counts: true,
+ *     output array:
+ *       output_data: [1, 2, 4, 7, 8, 9]
+ *       output_index: [0, 0, 1, 2, 2, 5, 3, 4, 4]
+ *       output_counts: [2, 1, 2, 1, 2, 1] *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:6545]: 8

File: bangc-ops/mlu_op.h [hunk 10]
```diff
@@ -6558,36 +6714,34 @@ mluOpGetUniqueWorkspaceSize(mluOpHandle_t handle,
...
-    @endverbatim
+ *   @verbatim
+ *     Example 1:
+ *     input array:
+ *       input: [1, 1, 2, 4, 4, 9, 7, 8, 8]
+ *     param:
+ *       mode: MLUOP_UNSORT_FORWARD
+ *     output array:
+ *       output: [1, 2, 4, 9, 7, 8]
+ *       inverse_indices: [0, 0, 1, 2, 2, 3, 4, 5, 5] *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:6725]: 9

File: bangc-ops/mlu_op.h [hunk 11]
```diff
@@ -6653,13 +6807,11 @@ mluOpUnique_v2(mluOpHandle_t handle,
...
  * - The example of the gather_nd operation is as follows:
-     @verbatim
-     input two arrays both by 3 * 2 --> params: [[1., 2.], [3., 4.], [5., 6.]]
-
-     --> indices: [[-1, 0], [1, 1]]
-
-     output array by 2 --> output: [5., 4.]
-     @endverbatim
+ *   @verbatim
+ *   input two arrays both by 3 * 2 --> params: [[1., 2.], [3., 4.], [5., 6.]] *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:6811]: 10

File: bangc-ops/mlu_op.h [hunk 12]
```diff
@@ -6736,20 +6888,21 @@ mluOpGatherNd(mluOpHandle_t handle,
...
+ *  The shape of output: [4,4,4].
+ *  indices: [[0], [2]]
+ *  updates: [[[5, 5, 5, 5], [6, 6, 6, 6],
+ *             [7, 7, 7, 7], [8, 8, 8, 8]],
+ *            [[5, 5, 5, 5], [6, 6, 6, 6],
+ *             [7, 7, 7, 7], [8, 8, 8, 8]]] *
+ *  -->output: [[[5, 5, 5, 5], [6, 6, 6, 6], [7, 7, 7, 7], [8, 8, 8, 8]],
+ *              [[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]],
+ *              [[5, 5, 5, 5], [6, 6, 6, 6], [7, 7, 7, 7], [8, 8, 8, 8]],
+ *              [[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]]] *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:6901]: 11

File: bangc-ops/mlu_op.h [hunk 13]
```diff
@@ -6823,40 +6976,40 @@ mluOpScatterNd(mluOpHandle_t handle,
...
-   @endverbatim
+ * @verbatim
+ *  Example 1:
+ *  The shape of output: [4,4,4].
+ *  mode:    MLUOP_SCATTERND_UPDATE
+ *  indices: [[0], [0]]
+ *  input:   [[[1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1]],
+ *            [[1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1]],
+ *            [[1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1]],
+ *            [[1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1]]] *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:6987]: 12

File: bangc-ops/mlu_op.h [hunk 14]
```diff
@@ -7215,17 +7383,16 @@ mluOpGetTransposeWorkspaceSize(mluOpHandle_t handle,
...
-      output array by 2 * 3 --> output: [[1, 2, 3],
-                                         [4, 5, 6]]
-     @endverbatim
+ *   @verbatim
+ *    input array by 3 * 2 -->
+ *        input: [[1, 4],
+ *                [2, 5],
+ *                [3, 6]]
+ *    param:
+ *      dims: 2, permute: (1, 0), *
```
Comment (WangYuan-cn) [inline bangc-ops/mlu_op.h:7392]: 13

File: docs/bangc-docs/user_guide/2_update_history/index.rst [hunk 15]
```diff
@@ -8,25 +8,32 @@
...
+     + :ref:`mluOpDeformRoiPoolBackward`
+     + :ref:`mluOpDeformRoiPoolForward`
+     + :ref:`mluOpGatherNd`
+     + :ref:`mluOpGetIndicePairs`
+     + :ref:`mluOpIndiceConvolutionBackwardData`
+     + :ref:`mluOpIndiceConvolutionBackwardFilter`
      + :ref:`indice_convolution_forward`
+     + :ref:`mat_mul`
+     + :ref:`reduce`
      + :ref:`roi_align_rotated_backward`
```
Comment (WangYuan-cn) [inline docs/bangc-docs/user_guide/2_update_history/index.rst:26]: 14

**Review by Unireverse — APPROVED** [review]

**Review by LittleReal — APPROVED** [review]
LGTM

**Pull request merged**